### PR TITLE
Integrate external package

### PR DESCRIPTION
**Pull request by shlecturer**
I haven't really tested this, as pip wanted to install an insane amount of extra packages on my machine.

### CODE REVIEW

File: GuessGame.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from time import sleep
 import keyboard
```
Comment (shlecturer) [inline GuessGame.py:3]: You may consider adding a `requirements.txt` to the repository and include the package `keyboard` in it. 

Easiest way is to run `pip freeze > requirements.txt` and then remove any packages that shouldn't be in there. At the moment, only `keyboard` should stay.

File: GuessGame.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 from time import sleep
 import keyboard
 
+
```
Comment (shlecturer) [inline GuessGame.py:5]: Added a newline (PEP-8 requires 2 newlines before or after a class definition).

File: GuessGame.py [hunk 3]
```diff
@@ -20,15 +21,13 @@ def intro():
             "to a hint by pressing 'H'.")
         sleep(4)
         print("Then let's start and Good Luck!")
-        print("Ready to start? Type 'y' or 'n'")
+        print("Press <enter> to start or <escape> to quit.")
```
Comment (shlecturer) [inline GuessGame.py:24]: I didn't think `y` and `n` were relevant anymore?

File: GuessGame.py [hunk 4]
```diff
@@ -49,45 +48,46 @@ def main():
...
-                        break
-                    elif guesses > number:
-                        print("The number was too high")
-                        break
-                elif hint == "":
+        if count >= 2:
+            print("Do you want a hint? Press 'H' or 'Enter' to pass: ")
+            hint = False
+            while True:
+                if keyboard.is_pressed('enter'):
```
Comment (shlecturer) [inline GuessGame.py:57]: Why not use the keyboard here as well, I reckoned.

File: GuessGame.py [hunk 5]
```diff
@@ -49,45 +48,46 @@ def main():
...
-                if guesses < n1 or guesses > n2:
-                    print("Out of the range!")
-                    continue
-                else:
-                    break
-            except:
+            except ValueError:
                 print("Invalid guesses! Try again.")
+            else:
+                if n1 <= guesses <= n2:
```
Comment (shlecturer) [inline GuessGame.py:75]: `x <= a <= y` is a very nice Python construct.

File: GuessGame.py (hunk 4 above)
Comment (Eregn) [reply GuessGame.py:57]: I was thinking to use here as well, I just forgot to do it. Thanks to have notice!

**Pull request merged**